static/src/components/EndScreen.js

```diff
@@ -0,0 +1,176 @@
+        this.state = {
+            userID: null,
+            displayData: null,
+            fetch: true,
+            open: false,
+        };
+        if (this.props.game) {
+            this.state.userData = this.props.game.player;
+            this.state.allPlayers = this.props.game.allPlayers;
+            this.getData();
+        }
+    }
+
+    componentDidMount() {
+        console.log('componentDidMount');
+    }
+
+    getData() {
+        if (this.state.fetch) {
+            const token = localStorage.getItem('token');
+            fetch(`/api/game/status?player_id=${this.state.displayID}`, {
+                method: 'get',
+                credentials: 'include',
+                headers: {
+                    'Accept': 'application/json', // eslint-disable-line quote-props
+                    'Content-Type': 'application/json',
+                    Authorization: token,
+                },
+            })
+            .then(response => response.json())
+            .then((body) => {
+                console.log('getData(body): ', body);
+                if (body.game) {
+                    this.setState({
+                        displayData: body.game.player,
+                        Players: body.players,
+                        allPlayers: body.game.allPlayers,
+                        fetch: false,
+                    });
+                }
+            });
+        }
+    }
+    handleOpen = () => {
+        this.setState({ open: true });
+    }
+
+    handleClose = () => {
+        this.setState({ open: false });
+    }
+    render() {
+        const primary = true;
+        const scoreBoardActions = [
+            <FlatButton
+                label='Close'
+                primary={true}
+                onClick={this.handleClose}
+            />,
+        ];
+        const rows = this.state.data.map(player =>
+            <PlayerRow key={player.id} data={player} />);
+        return (
+            <div>
+                <Dialog
+                    title='Score Board'
+                    actions={scoreBoardActions}
+                    modal={false}
+                    open={this.state.open}
+                    onRequestClose={this.handleClose}
+                >
+                    <Table>
+                        <TableHeader>
+                            <TableRow>
+                                <tableHeaderColumn> ID </tableHeaderColumn>
+                                <tableHeaderColumn> Points </tableHeaderColumn>
+                                <tableHeaderColumn> Coin </tableHeaderColumn>
+                                <tableHeaderColumn> Military </tableHeaderColumn>
+                            </TableRow>
+                        </TableHeader>
+                        <TableBody>
+                            {rows}
+                        </TableBody>
+                    </Table>
+                </Dialog>
+            </div>
+        );
+    }
+}
+const PlayerRow = (game) => {
```

Comment:
Each row should be the number of players in the game not fixed as the number of players will change